src/main/java/com/formulasearchengine/mathmlsim/similarity/MathPlag.java

```diff
@@ -0,0 +1,107 @@
+package com.formulasearchengine.mathmlsim.similarity;
+
+import com.formulasearchengine.mathmlsim.similarity.node.MathNode;
+import com.formulasearchengine.mathmlsim.similarity.node.MathNodeGenerator;
+import com.formulasearchengine.mathmlsim.similarity.result.Match;
+import com.formulasearchengine.mathmltools.mml.CMMLInfo;
+import org.apache.log4j.Logger;
+
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.xpath.XPathExpressionException;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * This class provides methods to compare mathematical expression in
+ * MathML format. The input can either be as a string or {@link CMMLInfo}
+ * object. In the first case the string will converters into the latter.
+ * <br/>
+ * At the moment this is purely a utility class and therefore
+ * has no public constructor. All methods are static.
+ *
+ * @author Vincent Stange
+ */
+public class MathPlag {
+
+    private static Logger logger = Logger.getLogger(MathPlag.class);
+
+    private MathPlag() {
+        // not visible, utility class only
+    }
+
+    /**
+     * Compare two CMMLInfo document against each other. The type defines how a Match will be displayed.
+     *
+     * @param refMathML  Reference MathML string (must contain pMML and cMML)
+     * @param compMathML Comparison MathML string (must contain pMML and cMML)
+     * @return list of matches / similarities, list can be empty.
+     */
+    public static List<Match> compareIdenticalMathML(String refMathML, String compMathML) throws IOException, ParserConfigurationException {
+        return compareDocuments(new CMMLInfo(refMathML), new CMMLInfo(compMathML), "identical");
+    }
+
+    /**
+     * Compare two CMMLInfo document against each other. The type defines how a Match will be displayed.
+     *
+     * @param refDoc  Reference CMMLInfo document
+     * @param compDoc Comparison CMMLInfo document
+     * @param type    defines how a Match will be displayed (similar / identical)
```

Comment:
Good point. Already used an enum but somehow never used it in the upper classes.